homeassistant/helpers/template.py

```diff
@@ -2095,15 +2095,23 @@ def today_at(hass: HomeAssistant, time_str: str = "") -> datetime:
     return datetime.combine(today, time_today, today.tzinfo)
 
 
-def relative_time(hass: HomeAssistant, value: Any) -> Any:
+
```

Comment:
Can you please remove this extra line#2098 this should fix your black failure.

Reply:
Well, it doesn’t appear that this patch will be approved, so I haven’t bothered.

Reply:
Thanks @rlippmann, I'm still following this issue and hoping it will be merged! Maybe @disforw can advise on next steps?